src/commands/edge_app_server.rs

```diff
@@ -13,9 +13,15 @@ use warp::{Filter, Rejection, Reply};
 
 pub const MOCK_DATA_FILENAME: &str = "mock-data.yml";
 
+#[derive(Debug, Clone, Deserialize)]
+pub enum Value {
+    Str(String),
+    Array(Vec<String>),
+}
+
 pub async fn run_server(
     path: &Path,
-    secrets: Vec<(String, String)>,
+    secrets: Vec<(String, Value)>,
```

Comment:
Hm - inst this strictly cli related format, may be pass it with string to parse later?

Reply:
yeah, I  also think it should all be in the server code.

Reply:
Done!